products/kubernetes-cluster-autoscaler.md

```diff
@@ -0,0 +1,126 @@
+---
+title: Kubernetes Cluster Autoscaler
+category: server-app
+tags: kubernetes
+permalink: /cluster-autoscaler
+versionCommand: |-
+  cluster-autoscaler --version
+releasePolicyLink: https://github.com/kubernetes/autoscaler/tree/master/cluster-autoscaler#releases
+changelogTemplate: https://github.com/kubernetes/autoscaler/releases/tag/cluster-autoscaler-__LATEST__
+releaseDateColumn: true
+eolColumn: true
+
+releases:
+  - releaseCycle: "1.32.0"
+    releaseDate: 2024-12-31
+    eol: false
```

Comment:
Considering "Each release is typically maintained as long as its corresponding Kubernetes version is supported", shouldn't eol be the same date as the corresponding Kubernetes version (documented on https://endoflife.date/kubernetes) ?